cli/src/commands/prev.rs

```diff
@@ -21,10 +21,15 @@ use crate::command_error::{user_error, CommandError};
 use crate::commands::next::choose_commit;
 use crate::ui::Ui;
 
-/// Move the working-copy commit to the parent revision
+/// Move the working-copy commit relative to the parent revision
 ///
 ///
-/// The command moves you to the parent in a linear fashion.
+/// The command moves `@-` (and therefore `@`) to an ancestor in a linear
+/// fashion. If you run `jj prev n` and `--edit` is not used or implied, then
+/// `jj prev` will operate as if `@` had been moved to its parent and then moved
+/// backwards by `n` revisions before running `jj new` to create a new commit.
+/// If `@-` resolves to multiple revisions, then the command fails.
```

Comment:
Use simpler language, theoreticals helps no one.

Reply:
I don't think there's a short answer. 

Here's a simplified sugggestion:

The command moves `@` to an ancestor relative to its parent. If you run `jj prev 2` and `--edit` isn't used, then `jj prev` operates as if you had done a `jj new` on the parent of the destination commit.

Reply:
But `@` is not moved to an ancestor, it ends up at a newly created commit on top of an ancestor. Or maybe I am thinking about `@` in the wrong way? It seems to me that `@-` is moved to an ancestor and a new `@` revision is created on top of it unless `--edit` is used. 😵‍💫

Reply:
> But `@` is not moved to an ancestor, it ends up at a newly created commit on top of an ancestor.

This is what the second sentence tries to explain. 

> Or maybe I am thinking about `@` in the wrong way?

I think of the working copy commit being always _attached_ to a parent, so my phrasing there is naturally going to reflect that.

Reply:
Stop using the word "move"! 
The symbol `@` represents the working copy, so don't explain it like you are moving `@` around. Say what it actually does!

Reply:
`@` is basically a pointer to the commit that's been materialized as the working copy.

I am saying `@` is moved to "x" as a shorthand for the commit "x" is materialized as the working copy.

Maybe you'd prefer `@` becomes "x"? Idk, do you have a preferred shorthand for describing this?

Re: Philip, every commit is attached to a parent except the repo root. I'm just hoping it can be a little clearer how the offset is translated to a target commit. It is an offset from `@-`, right? Maybe we can say the target is an ancestor `OFFSET` commits behind `@-`?

Reply:
> I am saying `@` is moved to "x" as a shorthand for the commit "x" is materialized as the working copy.
> 
> Maybe you'd prefer `@` becomes "x"? Idk, do you have a preferred shorthand for describing this?

I, here right now, understand it. But it is documentation, so it should be clear to the user (especially new users) what the command does. Nothing is moving, so don't say that. The `@` is a symbol for a concept, so don't use that. Explain the consequences and intent of the command.

Reply:
Ok, here is my new try with plain language and without referring to `@` or using the word move at all. Joy, I think you are right that it is clearer this way, but let me know what you think.

---

Change the working copy revision relative to the parent revision

The command creates a new empty working copy revision that is the child of
an ancestor OFFSET revisions behind the parent of the current working copy.

For example, when OFFSET is 1:

```text
D @      D
|/       |
A   =>   A @
|        |/
B        B
```

If `--edit` is passed, the working copy revision is changed to the parent of the current
working copy revision.

```text
D @  D
|/   |
C => @
|    |
B    B
|    |
A    A
```
If the working copy revision already has visible children, then `--edit` is
implied.

Reply:
Thanks for improving the documentation.